### PR TITLE
Hunting  smb quic rules

### PR DESCRIPTION
**Pull request by frack113**
### Summary of the Pull Request

Mount SMB over QUIC .

### Detailed Description of the Pull Request / Additional Comments

As I did not find a Dfir with it , I put them in the threat hunting.

### Example Log Event

`T1570_test_1_channel_Microsoft-Windows-PowerShell_Operational.json:{"SourceName":"Microsoft-Windows-PowerShell","ProviderGuid":"{a0c1853b-5c40-4b15-8766-3cf1c58f985a}","Level":"5","Keywords":"0x0","Channel":"Microsoft-Windows-PowerShell/Operational","Hostname":"Win11.lab.local","TimeCreated":"2023-07-21T05:14:28.089Z","@timestamp":"2023-07-21T05:14:28.089Z","EventID":4104,"Message":"Création du texte Scriptblock (1 sur 1) : \r\n\u0026 {New-SmbMapping -RemotePath \u0027\\\\example.com\\sales\u0027 -TransportType QUIC -SkipCertificateCheck\ncopy \u0027C:\\path\\to\\file.txt\u0027 \u0027Z:\\\u0027}\r\n\r\nID Scriptblock : 5ff77fae-d25a-4b89-83b1-7afa69a07431\r\nChemin d\u0027accès : ","Task":"2","MessageNumber":"1","MessageTotal":"1","ScriptBlockText":"\u0026 {New-SmbMapping -RemotePath \u0027\\\\example.com\\sales\u0027 -TransportType QUIC -SkipCertificateCheck\ncopy \u0027C:\\path\\to\\file.txt\u0027 \u0027Z:\\\u0027}","ScriptBlockId":"5ff77fae-d25a-4b89-83b1-7afa69a07431","Path":null}
T1570_test_1_channel_Microsoft-Windows-PowerShell_Operational.json:{"SourceName":"Microsoft-Windows-PowerShell","ProviderGuid":"{a0c1853b-5c40-4b15-8766-3cf1c58f985a}","Level":"5","Keywords":"0x0","Channel":"Microsoft-Windows-PowerShell/Operational","Hostname":"Win11.lab.local","TimeCreated":"2023-07-21T05:14:28.129Z","@timestamp":"2023-07-21T05:14:28.129Z","EventID":4104,"Message":"Création du texte Scriptblock (1 sur 1) : \r\n{New-SmbMapping -RemotePath \u0027\\\\example.com\\sales\u0027 -TransportType QUIC -SkipCertificateCheck\ncopy \u0027C:\\path\\to\\file.txt\u0027 \u0027Z:\\\u0027}\r\n\r\nID Scriptblock : d1aa82ea-64da-4944-b819-b30181c3e701\r\nChemin d\u0027accès : ","Task":"2","MessageNumber":"1","MessageTotal":"1","ScriptBlockText":"{New-SmbMapping -RemotePath \u0027\\\\example.com\\sales\u0027 -TransportType QUIC -SkipCertificateCheck\ncopy \u0027C:\\path\\to\\file.txt\u0027 \u0027Z:\\\u0027}","ScriptBlockId":"d1aa82ea-64da-4944-b819-b30181c3e701","Path":null}`

`T1570_test_2_channel_Microsoft-Windows-Sysmon_Operational.json:{"SourceName":"Microsoft-Windows-Sysmon","ProviderGuid":"{5770385f-c22a-43e0-bf4c-06f5698ffbd9}","Level":"4","Keywords":"0x8000000000000000","Channel":"Microsoft-Windows-Sysmon/Operational","Hostname":"Win11.lab.local","TimeCreated":"2023-07-21T05:16:56.369Z","@timestamp":"2023-07-21T05:16:56.369Z","EventID":1,"Message":"Process Create:\r\nRuleName: technique_id=T1018,technique_name=Remote System Discovery\r\nUtcTime: 2023-07-21 15:16:56.329\r\nProcessGuid: {2404497a-a168-64ba-8107-000000000400}\r\nProcessId: 5220\r\nImage: C:\\Windows\\System32\\net.exe\r\nFileVersion: 10.0.22621.1 (WinBuild.160101.0800)\r\nDescription: Net Command\r\nProduct: Microsoft® Windows® Operating System\r\nCompany: Microsoft Corporation\r\nOriginalFileName: net.exe\r\nCommandLine: \"C:\\WINDOWS\\system32\\net.exe\" USE * \\\\example.com\\sales /TRANSPORT:QUIC /SKIPCERTCHECK\r\nCurrentDirectory: C:\\Users\\admin\\AppData\\Local\\Temp\\\r\nUser: LAB\\admin\r\nLogonGuid: {2404497a-7c47-64ba-94db-4e0000000000}\r\nLogonId: 0x4EDB94\r\nTerminalSessionId: 1\r\nIntegrityLevel: High\r\nHashes: SHA1=76866DDE54EE3FA5BC8EFEFB9D44E6BF859973AA,MD5=BB1AE49B6B7C53499E94613761A6AC56,SHA256=AFBE51517092256504F797F6A5ABC02515A09D603E8C046AE31D7D7855568E91,IMPHASH=D45C37A5C97135204AD6E116C34946C3\r\nParentProcessGuid: {2404497a-a165-64ba-7f07-000000000400}\r\nParentProcessId: 10028\r\nParentImage: C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe\r\nParentCommandLine: \"powershell.exe\" \u0026 {NET USE * \u0027\\\\example.com\\sales\u0027 /TRANSPORT:QUIC /SKIPCERTCHECK\ncopy \u0027C:\\path\\to\\file.txt\u0027 \u0027*:\\\u0027}\r\nParentUser: LAB\\admin","Task":"1","RuleName":"technique_id=T1018,technique_name=Remote System Discovery","UtcTime":"2023-07-21 15:16:56.329","ProcessGuid":"{2404497a-a168-64ba-8107-000000000400}","ProcessId":"5220","Image":"C:\\Windows\\System32\\net.exe","FileVersion":"10.0.22621.1 (WinBuild.160101.0800)","Description":"Net Command","Product":"Microsoft® Windows® Operating System","Company":"Microsoft Corporation","OriginalFileName":"net.exe","CommandLine":"\"C:\\WINDOWS\\system32\\net.exe\" USE * \\\\example.com\\sales /TRANSPORT:QUIC /SKIPCERTCHECK","CurrentDirectory":"C:\\Users\\admin\\AppData\\Local\\Temp\\","User":"LAB\\admin","LogonGuid":"{2404497a-7c47-64ba-94db-4e0000000000}","LogonId":"0x4edb94","TerminalSessionId":"1","IntegrityLevel":"High","Hashes":"SHA1=76866DDE54EE3FA5BC8EFEFB9D44E6BF859973AA,MD5=BB1AE49B6B7C53499E94613761A6AC56,SHA256=AFBE51517092256504F797F6A5ABC02515A09D603E8C046AE31D7D7855568E91,IMPHASH=D45C37A5C97135204AD6E116C34946C3","ParentProcessGuid":"{2404497a-a165-64ba-7f07-000000000400}","ParentProcessId":"10028","ParentImage":"C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe","ParentCommandLine":"\"powershell.exe\" \u0026 {NET USE * \u0027\\\\example.com\\sales\u0027 /TRANSPORT:QUIC /SKIPCERTCHECK\ncopy \u0027C:\\path\\to\\file.txt\u0027 \u0027*:\\\u0027}","ParentUser":"LAB\\admin"}
`
### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules-threat-hunting/windows/powershell/powershell_script/posh_ps_netshare_quic.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+title: SMB over QUIC Via PowerShell Script
+id: 6df07c3b-8456-4f8b-87bb-fe31ec964cae
+related:
+    - id: 2238d337-42fb-4971-9a68-63570f2aede4
+      type: similar
+status: experimental
+description: Detects mount windows SMB over QUIC
```
Comment (phantinuss) [inline rules-threat-hunting/windows/powershell/powershell_script/posh_ps_netshare_quic.yml:7]:
```suggestion
description: Detects the mounting of Windows SMB over QUIC which is not expected in an enterprise environment
```

File: rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+title: SMB over QUIC Via Net.EXE
+id: 2238d337-42fb-4971-9a68-63570f2aede4
+related:
+    - id: 6df07c3b-8456-4f8b-87bb-fe31ec964cae
+      type: similar
+status: experimental
+description: Detects mount windows SMB over QUIC
```
Comment (phantinuss) [inline rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml:7]: Detects the mounting of Windows SMB over QUIC which is not expected in an enterprise environment

**Review by phantinuss — APPROVED** [review]

File: rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml (hunk 2 above)
Comment (nasbench) [inline rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml:7]:
```suggestion
description: Detects the mounting of Windows SMB over QUIC which is not expected in an enterprise environment
```

File: rules-threat-hunting/windows/powershell/powershell_script/posh_ps_netshare_quic.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+title: SMB over QUIC Via PowerShell Script
+id: 6df07c3b-8456-4f8b-87bb-fe31ec964cae
+related:
+    - id: 2238d337-42fb-4971-9a68-63570f2aede4
+      type: similar
+status: experimental
+description: Detects the mounting of Windows SMB over QUIC which is not expected in an enterprise environment
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/74438b0237d141ee9c99747976447dc884cb1a39/atomics/T1570/T1570.md
```
Comment (nasbench) [inline rules-threat-hunting/windows/powershell/powershell_script/posh_ps_netshare_quic.yml:9]:
```suggestion
    - https://github.com/redcanaryco/atomic-red-team/blob/74438b0237d141ee9c99747976447dc884cb1a39/atomics/T1570/T1570.md
    - https://learn.microsoft.com/en-us/powershell/module/smbshare/new-smbmapping?view=windowsserver2022-ps
    - https://www.trustedsec.com/blog/making-smb-accessible-with-ntlmquic/
```

File: rules-threat-hunting/windows/powershell/powershell_script/posh_ps_netshare_quic.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+title: SMB over QUIC Via PowerShell Script
+id: 6df07c3b-8456-4f8b-87bb-fe31ec964cae
+related:
+    - id: 2238d337-42fb-4971-9a68-63570f2aede4
+      type: similar
+status: experimental
+description: Detects the mounting of Windows SMB over QUIC which is not expected in an enterprise environment
```
Comment (nasbench) [inline rules-threat-hunting/windows/powershell/powershell_script/posh_ps_netshare_quic.yml:7]:
```suggestion
description: Detects the mounting of Windows SMB shares over QUIC. Which can be an unexpected event in some enterprise environments
```

File: rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    - attack.lateral_movement
+    - attack.t1570
+    - detection.threat_hunting
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        - Image|endswith: '\net.exe'
+        - OriginalFileName: 'net.exe'
```
Comment (nasbench) [inline rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml:22]:
```suggestion
        - Image|endswith:
            - '\net.exe'
            - '\net1.exe'
        - OriginalFileName:
            - 'net.exe'
            - 'net1.exe'
```

File: rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml [hunk 6]
```diff
@@ -0,0 +1,28 @@
+title: SMB over QUIC Via Net.EXE
+id: 2238d337-42fb-4971-9a68-63570f2aede4
+related:
+    - id: 6df07c3b-8456-4f8b-87bb-fe31ec964cae
+      type: similar
+status: experimental
+description: Detects the mounting of Windows SMB over QUIC which is not expected in an enterprise environment
```
Comment (nasbench) [inline rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml:7]:
```suggestion
description: Detects the mounting of Windows SMB shares over QUIC. Which can be an unexpected event in some enterprise environments.
```

File: rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/74438b0237d141ee9c99747976447dc884cb1a39/atomics/T1570/T1570.md
```
Comment (nasbench) [inline rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml:9]:
```suggestion
    - https://github.com/redcanaryco/atomic-red-team/blob/74438b0237d141ee9c99747976447dc884cb1a39/atomics/T1570/T1570.md
    - https://www.trustedsec.com/blog/making-smb-accessible-with-ntlmquic/
```

**Review by nasbench — APPROVED** [review]

File: rules-threat-hunting/windows/powershell/powershell_script/posh_ps_new_smbmapping_quic.yml [hunk 8]
```diff
@@ -0,0 +1,30 @@
+title: SMB over QUIC Via PowerShell Script
+id: 6df07c3b-8456-4f8b-87bb-fe31ec964cae
+related:
+    - id: 2238d337-42fb-4971-9a68-63570f2aede4
+      type: similar
+status: experimental
+description: Detects the mounting of Windows SMB shares over QUIC. Which can be an unexpected event in some enterprise environments
```
Comment (phantinuss) [inline rules-threat-hunting/windows/powershell/powershell_script/posh_ps_new_smbmapping_quic.yml:7]:
```suggestion
description: Detects the mounting of Windows SMB shares over QUIC, which can be an unexpected event in some enterprise environments
```

File: rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml [hunk 9]
```diff
@@ -0,0 +1,33 @@
+title: SMB over QUIC Via Net.EXE
+id: 2238d337-42fb-4971-9a68-63570f2aede4
+related:
+    - id: 6df07c3b-8456-4f8b-87bb-fe31ec964cae
+      type: similar
+status: experimental
+description: Detects the mounting of Windows SMB shares over QUIC. Which can be an unexpected event in some enterprise environments.
```
Comment (phantinuss) [inline rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml:7]:
```suggestion
description: Detects the mounting of Windows SMB shares over QUIC, which can be an unexpected event in some enterprise environments.
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**